### DISCUSSION TITLE
Dịch NP-hard, NP-complete

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#78] [Terminology] [answered]
Hai từ này dịch thành `NP-khó` và `NP-đầy đủ` được không nhỉ?

cc @mlbvn/handson-ml2

**Comment by ngcthuong** [comment]
NP-hard --> NP-khó: Ok, 
NP complete --> NP-hoàn chỉnh?

**Reply by quangvnai** [reply]
khó và đầy đủ là thuật ngữ phổ biến dùng trước nay r anh ạ

**Reply by ngcthuong** [reply]
Ok. Nếu trước dùng vậy rồi mình vẫn dùng lại thôi.

**Comment by minhduc0711** [comment, marked as answer]
Vậy dịch là  `NP-khó` và `NP-đầy đủ` nhé mọi người.
